registry/extension/oci/oci.go

```diff
@@ -95,9 +95,28 @@ func (d *ociNamespace) GetRepositoryRoutes() []extension.Route {
 }
 
 // GetRegistryRoutes returns a list of extension routes scoped at a registry level
-// There are no registry scoped routes exposed by this namespace
 func (d *ociNamespace) GetRegistryRoutes() []extension.Route {
-	return nil
+	var routes []extension.Route
+
+	if d.discoverEnabled {
```

Comment:
Same as the previous case, why d? The struct is called ociNamespace so for consistency use o or on

Reply:
renamed to 'o'